src/style.css

```
@@ -1,3 +1,77 @@
 body {
   margin: 0;
+  background-color: grey;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+header {
```

Use the class selector for header
```suggestion
.header {
```